src/components/tasks/card/index.tsx

```diff
@@ -324,23 +282,50 @@ const Card: FC<Props> = ({
         </div>
     );
 
-    const ProgressIndicator = () => (
-        <div className={classNames.progressIndicator}>
-            <div
-                className={`
-                ${handleProgressColor(
-                    content.percentCompleted,
-                    content.startedOn,
-                    content.endsOn
-                )}
-                ${classNames.progressStyle}
-                `}
-                style={{
-                    width: `${content.percentCompleted}%`,
-                }}
-            ></div>
-        </div>
-    );
+    const handleProgressUpdate = () => {
+        if (
+            content.assignee === data?.username ||
+            data?.roles.super_user === true
+        ) {
+            setProgress(true);
+        } else {
+            toast(ERROR, 'You cannot update progress');
+        }
+    };
+
+    const debounceSlider = (debounceTimeOut: number) => {
+        if (debounceTimeOut) {
+            clearTimeout(debounceTimeOut);
+        }
+        const timer = setTimeout(() => {
+            handleSliderChangeComplete(cardDetails.id, progressValue);
+        }, 1000);
+        setDebounceTimeOut(Number(timer));
+    };
```

Comment:
- do we not have a `debounce` utility? if not should we move this util function to a `utils` folder?

Reply:
I had a discussion with @vvaibhavdesai, we have agreed to revamp card component in which I will be fixing things, currently in this PR, I am not planning to do so in this PR.

Reply:
- please create a ticket for this and link it to this pr.

Reply:
it's already created
https://github.com/Real-Dev-Squad/website-status/issues/687

Reply:
https://github.com/Real-Dev-Squad/website-status/issues/687